ui/src/holdingpen-new/containers/DetailPageContainer/AuthorDetailPageContainer.tsx

```diff
@@ -100,17 +103,14 @@ const AuthorDetailPageContainer: React.FC<AuthorDetailPageContainerProps> = ({
                       <b>Status:</b> {data?.get('status')}
                     </p>
                   )}
-                  {data?.getIn(['acquisition_source', 'orcid']) && (
+                  {authorOrcid && (
                     <p className="mb0">
                       <b>ORCID:</b>{' '}
                       <a
-                        href={`https://orcid.org/my-orcid?orcid=${data?.getIn([
-                          'acquisition_source',
-                          'orcid',
-                        ])}`}
+                        href={`https://orcid.org/my-orcid?orcid=${authorOrcid}`}
                         target="_blank"
                       >
-                        {data?.getIn(['acquisition_source', 'orcid'])}
+                        {authorOrcid}
```

Comment:
Can we reuse https://github.com/inspirehep/inspirehep/blob/9a7b03c9f6ed36681d43300553305d697d35ab74/ui/src/holdingpen-new/components/Links.tsx#L31 ?